examples/deadline/Local-Zone/python/README.md

```diff
@@ -0,0 +1,72 @@
+
+    ```bash
+    pip install -r requirements.txt
+    ```
+
+3.  If working on the `release` branch, this step can be skipped. If working on `mainline`, navigate to the base directory where the build and packaging scripts are, then run them and install the result over top of the `aws-rfdk` version that was installed in the previous step:
+    ```bash
+    # Navigate to the root directory of the RFDK repository
+    pushd ../../../..
+    # Enter the Docker container to run the build and pack scripts
+    ./scripts/rfdk_build_environment.sh
+    ./build.sh
+    ./pack.sh
+    # Exit the Docker container
+    exit
+    # Navigate back to the example directory
+    popd
+    pip install ../../../../dist/python/aws-rfdk-<version>.tar.gz
+    ```
+
+4.  You must read and accept the [AWS Thinkbox End-User License Agreement (EULA)](https://www.awsthinkbox.com/end-user-license-agreement) to deploy and run Deadline. To do so, change the value of the `accept_aws_thinkbox_eula` in `package/lib/config.py` to `ACCEPTS` like this:
+
+    ```py
+    self.accept_aws_thinkbox_eula: AwsThinkboxEulaAcceptance = AwsThinkboxEulaAcceptance.USER_ACCEPTS_AWS_THINKBOX_EULA
+    ```
+
+5. Change the value of the `deadline_version` variable in `package/lib/config.py` to specify the desired version of Deadline to be deployed to your render farm. RFDK is compatible with Deadline versions 10.1.9.x and later. To see the available versions of Deadline, consult the [Deadline release notes](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/release-notes.html). It is recommended to use the latest version of Deadline available when building your farm, but to pin this version when the farm is ready for production use. For example, to pin to the latest `10.1.13` release of Deadline, use `10.1.13.1`.
+
+6. Change the value of the `deadline_client_linux_ami_map` variable in `package/config.py` to include the region + AMI ID mapping of your EC2 AMI(s) with Deadline Worker.
+
+7. Also in `package/lib/config.py`, you can set the `availability_zones_standard` and `availability_zones_local` values to the availability zones you want to use. These values must all be from the same region. It's recommended you use at least two standard zones, but you can use more if you'd like. For the local zones you can use as many as you'd like.
+
+8. Due to limitations of the Python implementation of CDK, you'll need to manually modify the availability zones in the CDK context file so your VPC deploys where you want it. In the TypeScript version we accomplish this by overriding `availabilityZones()` on the network tier stack, but this doesn't work in Python. Instead, either create or modify `cdk.context.json` to have an entry like the following, note you need to replace `<account #>` with your AWS account number. The example uses us-west-2, so the key includes this, but if you were to use a different region you'd also have to modify it in the key.
```

Comment:
afaik, the `cdk.context.json` file only gets created after doing a synth, no? If so, then there's a step missing where they'll have to do a synth to generate the file so that they can modify it.

Reply:
@horsmand I just noticed this section of the CDK's VPC docs: https://docs.aws.amazon.com/cdk/api/latest/docs/aws-ec2-readme.html#control-over-availability-zones

Have you tried the AZ override that they suggest (overriding the Stack's `availabilityZones` property)?

Reply:
I've tried overriding the `availabilityZones()` method as they mention there (this worked in the TypeScript app) as well as the property itself, but it's immutable.

I didn't try `construct.node.setContext()`, I'll give that a go.

Reply:
Should be possible in Python as well, though the syntax will be a hair more complex. AFAIK, it's something like:
```python
@property
def availability_zones():
  return [ ... <whatever> ... ]
```

Reply:
Well I thought I tried overriding the `availability_zones` method every way possible on my first attempt, but it seems I was able to get it to work by copying the signature exactly:
```
@builtins.property # type: ignore
    @jsii.member(jsii_name="availabilityZones")
    def availability_zones(self) -> typing.List[builtins.str]:
```

Reply:
This is now removed. Resolving